TDP-4050 document recording/transcription deletion (#516)
Bandwidth/bandwidth.github.io: voice/methods/recordings/deleteCallsCallIdRecordingsRecordingIdTranscription.md

|:----------|:------------|
| None      | None        |

> Note: After the deletion is requested and a `204` is returned, the transcription will not be accessible anymore. However, the transcription is not deleted immediately. The deletion process can take from 24 to 48 hours after it was requested.

Reviewer: ```suggestion
Note: After the deletion is requested and a `204` is returned, the transcription will not be accessible anymore. However, it is not deleted immediately. This deletion process, while transparent and irreversible, can take an additional 24 to 48 hours.
```
